app/src/main/java/de/tum/in/tumcampusapp/utils/NetUtils.java

```diff
@@ -119,38 +116,6 @@ public static boolean isConnectedWifi(Context context) {
             Utils.log(e);
             return Optional.absent();
         }
-
-    }
-
-    /**
-     * Download a file in the same thread.
-     * If file already exists the method returns immediately
-     * without downloading anything
-     *
-     * @param url    Download location
-     * @param target Target filename in local file system
-     * @throws IOException When the download failed
-     */
-    @Deprecated
-    public void downloadToFile(String url, String target) throws IOException {
```

Comment:
Good riddance. Together with #1027 we can probably completely get rid of NetUtils